.github/workflows/main.yml

```
@@ -32,24 +32,24 @@ jobs:
-        
-            # Print the contents of release_notes.txt to verify
-            echo "Contents of release_notes.txt:"
-            cat release_notes.txt
-        
-            # Set the release notes as an environment variable
-            echo "release-notes-github<<EOF" >> $GITHUB_ENV
-            echo "$release_notes" >> $GITHUB_ENV
-            echo "EOF" >> $GITHUB_ENV
+          # Extract release notes from CHANGELOG.md
+          release_notes=$(awk '/^# v/{count++} count<=1' CHANGELOG.md | tail -n +3 | sed 's/^##/**/; s/$/**/')
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)